apps/afisha/admin/events.py

```diff
@@ -121,7 +121,7 @@ def icon(status, lable):
 
     @admin.display(boolean=True, description="Платное")
     def is_paid(self, obj):
-        if obj.action == "TICKETS":
+        if obj.action_text == "TICKETS":
```

Comment:
Может значение из ActionType.choices выбирать? как никак красивее код будет. А заодно при определения самих свойств в моделе.
Кстати, в базе будет храниться TICKETS? Может хранить слова как на русском языке? Мы же фронту отправляем заголовок на русском

Reply:
Я не уверен чем будет лучше использование ActionType.choices. Либо я чего-то не понял )

По поводу БД - да, сейчас там хранятся ключи TICKETS, REGISTRATION, STREAM. Переделать вид, в котором они хранятся, наверное, можно, но вопрос нужно ли. У нас на самом деле в этом плане мешанина в проекте: в каких-то чойс филдах мы возвращаем ключи, в каких-то значения.

Reply:
я бы вообще одной строкой тут все записал

```suggestion
        return obj.action_text == "TICKETS"
```

Reply:
Пофигу на значения которые есть сейчас в базе, этого ещё нет на проде. Почему я за сами значения? потому что я хотел бы не заставлять привязываться админов к этим трём полям, если мы будем хранить там значения, то они в любой момент смогут ввести (в случае если мы дропдауну разрешим дополнительные значения принимать от пользователя, а делается это в одну строку на js) и всё будет работать. А вот если мы оставим как есть, то придётся потом через миграции править базу и всё равно сделать так же.
А у меня есть желание дать задачку, в которой сделать правку руками, я про это говорил ещё при первоначальном создании этого функционала, так как я уверен, на трёх они не остановятся.

Reply:
Хм, ну тут согласен. Есть большая вероятность, что тремя полями мы не обойдемся и будет неплохо дать возможность добавлять поле админам.
Я не знаю как ты предлагаешь это решить через js, вообще не шарю в этом ) Но в качестве альтернативного варианта можно наверное сделать отдельную модель, в которой будут храниться, добавляться, изменяться значения полей и оттуда тянуть информацию о доступных вариантах.